components/mail_and_packages.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "mail_and_packages",
+    "owner": ["@moralmunky", "@firstof9"],
+    "manifest": "https://raw.githubusercontent.com/moralmunky/Home-Assistant-Mail-And-Packages/0.3.0/custom_components/mail_and_packages/manifest.json",
```

Comment:
This should point to your development branch, not a specific tag.

Reply:
0.3.0 is the current dev branch.

Reply:
Yeah, was default branch i meant

Reply:
So the manifest needs to be under the default branch not the dev branch or is this ok?

Reply:
The one that will keep being updated, i highly doubt that is 0.0.3

Reply:
Alright thanks ludeeus, I'll fix it up once we cut a new latest.